doc/03_reference/verification.rst

```diff
@@ -89,20 +89,16 @@ Prerequisites & Environment Setup
 
 In order to run the co-simulation flow, you'll need:
 
-- A SystemVerilog simulator that supports UVM. The flow is currently
-  tested with VCS.
+- A SystemVerilog simulator that supports UVM. The flow is currently tested with VCS.
 
 - A RISC-V instruction set simulator, such as Spike_ or OVPsim_.
-  Note that Spike must be configured with
-  ``--enable-commitlog`` and ``--enable-misaligned``. The commit log
-  is needed to track the instructions that were executed and
-  ``--enable-misaligned`` tells Spike to simulate a core that
-  handles misaligned accesses in hardware (rather than jumping to a
-  trap handler).
-  In addition, Spike does not support the `RISC-V Bit Manipulation Extension <bitmanip_>`_  (Bitmanip) by default.
-  To support this draft extension implemented in Ibex, the `riscv-bitmanip branch <Spike_>`_ of Spike needs to be used.
-  If it is desired to simulate the core with the Icache enabled, a lowRISC-specific branch
-  of Spike must be used, `found here <https://github.com/lowRISC/riscv-isa-sim/tree/ibex>`_.
+  Note that when building Spike the ``--enable-commitlog`` and ``--enable-misaligned`` options must be passed to the ``configure`` script.
+  ``--enable-commitlog`` is needed to produce log output to track the instructions that were executed.
+  ``--enable-misaligned`` tells Spike to simulate a core that handles misaligned accesses in hardware (rather than jumping to a trap handler).
+  If it is desired to simulate the core with the Icache enabled, a lowRISC-specific branch of Spike must be used, `found here <https://github.com/lowRISC/riscv-isa-sim/tree/ibex>`_.
+  If using Bitmanip note that Spike_ will be required as the simulator.
```

Comment:
nit: this isn't quite accurate since OVPsim natively supports the B-extension.
I think it's safe to remove this as there really is only a hard requirement to use Spike if we want to enable the Icache.

Reply:
Ah ok, better replace it with a note that OVPSim, like Spike master may be supporting a different version of the bitmanip specification to our RTL.

Reply:
yup, sgtm!